Makefile

```
@@ -462,12 +462,23 @@ dev.shell.%: ## Run a shell on the specified service's container.
 dev.dbshell:
 	docker compose exec mysql80 bash -c "mysql"
 
+dev.dbsetupmysql8:
+	docker compose exec -T mysql80 bash -e -c "mysql -uroot mysql" < provision-mysql80.sql
+
+dev.dbcleanmysql8:
+	docker container rm edx.devstack.mysql80
```

If we use docker compose, we can avoid the prefix:
```suggestion
	docker compose rm mysql80
```

---

Or I guess we could just use `make mysql80-remove-containers`.